docs/protocol/core/v3.0.rst

```diff
@@ -626,9 +716,9 @@ containing the following names:
 
 ``metadata_encoding``
 
-    A string containing the MIME type that has been used for encoding
+    A string containing the URI that has been used for encoding
     group and array metadata documents. If JSON encoding is used, this
-    value must be "application/json".
+    value must be ``"https://purl.org/zarr/spec/metadata/json/3.0"``.
```

Comment:
In general we want URIs to dereference to some documentation that defines the metadata encoding. But for JSON, I don't think we need to create a new URI, we could just point to an existing URI. E.g., we could use the JSON spec URI: `https://tools.ietf.org/html/rfc8259`.

Reply:
That is fine with me, but one of the reason we discuss to use URI instead of mimetype was the the same mimetype can encore metadata in a different way. So pointing to the JSON spec URI seem to me like it has the same issue.

You do would to not only point to the format of the file, but also it's structure; At least that was my understanding.